test/parallel_api/dynamic_selection/sycl/test_dynamic_load_policy_sycl.pass.cpp

```diff
@@ -53,37 +56,37 @@ main()
     auto n = u.size();
 
     //If building the universe is not a success, return
-    if (n == 0)
-        return 0;
+    if (n != 0)
+    {
+        // should be similar to round_robin when waiting on policy
+        auto f = [u, n](int i) { return u[i % u.size()]; };
 
-    // should be similar to round_robin when waiting on policy
-    auto f = [u, n](int i) { return u[i % u.size()]; };
+        auto f2 = [u, n](int i) { return u[0]; };
+        // should always pick first when waiting on sync in each iteration
 
-    auto f2 = [u, n](int i) { return u[0]; };
-    // should always pick first when waiting on sync in each iteration
+        constexpr bool just_call_submit = false;
+        constexpr bool call_select_before_submit = true;
 
-    constexpr bool just_call_submit = false;
-    constexpr bool call_select_before_submit = true;
-    if (test_dl_initialization(u) || test_select<policy_t, decltype(u), decltype(f2)&, false>(u, f2) ||
-        test_submit_and_wait_on_event<just_call_submit, policy_t>(u, f2) ||
-        test_submit_and_wait_on_event<call_select_before_submit, policy_t>(u, f2) ||
-        test_submit_and_wait<just_call_submit, policy_t>(u, f2) ||
-        test_submit_and_wait<call_select_before_submit, policy_t>(u, f2) ||
-        test_submit_and_wait_on_group<just_call_submit, policy_t>(u, f) ||
-        test_submit_and_wait_on_group<call_select_before_submit, policy_t>(u, f)
+        auto actual = test_dl_initialization(u);
+        EXPECT_EQ(0, actual, "");
```

Comment:
Let's enumerate messages as "#1", "#2" and so far and so on. Or insert some text.

Reply:
The actual tests do already have the macros in them, so the additional macro checking the return code here is redundant. Those that do not have macros have a message and a return code.